src/gc/env/gcenv.os.h

```diff
@@ -7,6 +7,17 @@
 #ifndef __GCENV_OS_H__
 #define __GCENV_OS_H__
 
+#ifdef Sleep
+// This is a funny workaround for the fact that "common.h" defines Sleep to be
```

Comment:
Oh I see that you just moved it from somewhere else ... you can ignore my comment.